src/ui/TodoistQuery.svelte

```diff
@@ -76,6 +77,14 @@
     }
   });
 
+  async function callTaskModal() {
+    new CreateTaskModal(
+      app,
+      api,
+      true
+    );
+  }
```

Comment:
I don't think this function needs to be `async`, you could probably remove this (+ the corresponding `await` below).